lib/beautiful/theme_assets.lua

```diff
@@ -207,48 +208,57 @@ function theme_assets.wallpaper(bg, fg, alt_fg, s)
     return img
 end
 
---- Recolor unfocused titlebar icons.
--- @tparam table theme Beautiful theme table
+--- Recolor titlebar icons.
+-- @tparam table theme Beautiful theme table.
 -- @tparam color color Icons' color.
+-- @tparam string state `"normal"` or `"focus"`.
+-- @tparam string postfix `nil`, `"hover"` or `"press"`.
```

Comment:
Or alternatively: Shouldn't the code insert a `_` between the postfix and the rest of the string?